x-pack/solutions/search/plugins/enterprise_search/server/deprecations/index.test.ts

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CloudSetup } from '@kbn/cloud-plugin/server';
+
+let mockedFetchConnectors = jest.fn();
+jest.mock('@kbn/search-connectors', () => {
+  return {
+    __esModule: true,
+    ...jest.requireActual('@kbn/search-connectors'),
+    fetchConnectors: () => mockedFetchConnectors(),
+  };
+});
+import { GetDeprecationsContext } from '@kbn/core-deprecations-server';
+
+import { Connector } from '@kbn/search-connectors';
+
+import { ConfigType } from '..';
+
+import {
+  getCrawlerDeprecations,
+  getEnterpriseSearchNodeDeprecation,
+  getNativeConnectorDeprecations,
+} from '.';
+
+const ctx = {
+  esClient: {
+    asInternalUser: {},
+  },
+} as GetDeprecationsContext;
+const cloud = { baseUrl: 'cloud.elastic.co', deploymentId: '123', cloudId: 'abc' } as CloudSetup;
+const notCloud = {} as CloudSetup;
+const docsUrl = 'example.com';
+describe('Enterprise Search node deprecation', () => {
+  it('Tells you to remove capacity if running on cloud', () => {
+    const config = { host: 'example.com' } as ConfigType;
+    const deprecations = getEnterpriseSearchNodeDeprecation(config, cloud, docsUrl);
+    expect(deprecations).toHaveLength(1);
+    const steps = deprecations[0].correctiveActions.manualSteps;
+    expect(steps).toHaveLength(6);
+    const stepsStr = steps.join(', ');
+    expect(stepsStr).toMatch('Go to cloud.elastic.co');
```

Comment:
@seanstory Shouldn't this be 'Go to {baseUrl}'? 
The Ent Search nodes message has a different URL than the integration server message.

Reply:
fixed in https://github.com/elastic/kibana/pull/205500/commits/2fb81d0218e8d19c4b5c7622514421d7a3ce402b